modules/common/src/main/scala/io/circe/jackson/enrich/JacksonParser.scala

```diff
@@ -0,0 +1,40 @@
+package io.circe.jackson.enrich
```

Comment:
Is it a copy from circe jackson? It feels that in this case we can actually put our copyright as you're just impelmenting the interace?

Reply:
yes, it is copy